clusters/app.ci/release-controller/admin_deploy-ocp-publish-art.yaml

```diff
@@ -0,0 +1,1481 @@
+##################################################################################
```

Comment:
@jupierce can you add yourself (and probably @bradmwilliams ) in an OWNERS file in clusters/app.ci/release-controller so you don't need us for approval?

Reply:
Will do!